CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 21.10.4
+
+### Additions and Improvements
+- Removed deprecated hash variable `protected volatile Hash hash;` which was used for private transactions [#3110](https://github.com/hyperledger/besu/pull/3110)
```

Comment:
this should be under 21.10.3 since we haven't released that version yet (latest is 21.10.2)